Core/src/OffConverter/OffReader.cpp

```diff
@@ -68,7 +68,7 @@ std::shared_ptr<OffModel> OffReader::readFile(const std::string& filename)
 		int nrOfVertices, nrOfFaces, nrOfEdges;
 		nrOfVertices = lineData[0];
 		nrOfFaces = lineData[1];
-		nrOfEdges = lineData[2];
+		nrOfEdges = lineData[2]; //can be ignored
```

Comment:
How come we're ignoring the edges?

Reply:
The information about the number of edges isn't used at any other point in the code.
Furthermore de official .off documentation states that it is not neccessary and can be neglected.
Therefore a lot of .off files have set this number to 0 even though the number of edges is not 0.

Reply:
Right, I understand. After merging this PR, please put this as an enhancement issue.

Reply:
Okay